Makefile

```diff
@@ -32,11 +32,14 @@ SNAP=$(shell which snap)
 APT=$(shell which apt)
 WGET=$(shell which wget)
 RUST=$(shell which rustup)
+CROSS=$(shell which cross)
+DOCKER=$(shell which docker)
 
 DESTDIR = /
 INSTALL_PREFIX = usr/bin
 LIBRARY_PREFIX = usr/lib
 SHARE_PREFIX = usr/share
+RUST_BIN = ${HOME}/.cargo/bin
```

Comment:
idea: maybe CARGO_BIN is a naming which is closer to the reality?

Reply:
Made it CARGO_PREFIX to be consistent with the other prefixes.